stages/02_process_data.py

```diff
@@ -0,0 +1,32 @@
+# Imports
+# -------
+import os
+import gzip
+import pandas as pd
+
+_, _, files = next(os.walk('downloads'))
+files = [ os.path.join('downloads', file) for file in files ]
+
+os.makedirs('brick', exist_ok=True)
+
+for file in files:
+
+  out_file = os.path.join('brick', file.split('/')[1].split('.')[0] + '.parquet')
+
+  try:
+
+      if file.endswith('xlsx'):
+          reaction_data = pd.read_excel(file)
+          reaction_data.to_parquet(out_file)
+
+      elif file.endswith('gz'):
+          with gzip.open(file, 'rb') as file_in:
+              lines = file_in.read().decode("utf-8").split('\n')
+              data = [ line.split('\t') for line in lines ]
+              headers = data.pop(0)
+              df = pd.DataFrame(data, columns=headers)
```

Comment:
```python
df = pd.read_csv(file, sep='\t', encoding='utf-8')
```

Reply:
I did try that:

``ADRECS File Conversion Failed: Error converting column "BADD_A00003" to bytes using encoding UTF8. Original error: bad argument type for built-in operation```

I would get this.

Reply:
I might pass on that one.

Reply:
On which file?

Reply:
```
http://bioinf.xmu.edu.cn/ADReCS/download/v3.3/Drug_information_v3.3.xlsx
```

Reply:
That's odd, it's working here:

```python
In [15]: import pandas, openpyxl; ( pandas.__version__, openpyxl.__version__ )
Out[15]: ('2.2.2', '3.1.2')

In [16]: pd.read_excel('download/v3.3/Drug_information_v3.3.xlsx', engine='openpyxl').count()
Out[16]:
DRUG_ID          2601
DRUG_NAME        2601
DRUG_SYNONYMS    2601
DrugBank_ID      2601
PubChem_ID       2601
MESH_ID          2601
KEGG_ID          2601
TTD_ID           2601
dtype: int64
```

Reply:
Sorry wrong file: http://bioinf.xmu.edu.cn/ADReCS/download/v3.3/Drug_ADR_v3.3.txt.gz

Reply:
It was `ADReCS_ADR_Severity_Grade_v3.3.txt.gz` because of the comment below.